plugins/ern_v0.46.0+/react-native-image-picker_v3.0.0+/config.json

```diff
@@ -0,0 +1,51 @@
+{
+  "android": {
+    "root": "android",
+    "dependencies": [
+      "implementation 'androidx.core:core:1.3.1'",
```

Comment:
I don't think the `core` dependency needs to be listed here: It's a dependency of `appcompat` (which, as a dependency of `react-native`, is _always_ added).